general/g.tempfile/g.tempfile.html

```diff
@@ -1,19 +1,25 @@
 <h2>DESCRIPTION</h2>
 
 <em>g.tempfile</em>
-is designed for shell scripts that need to use large temporary files.
-GRASS provides a mechanism for temporary files that does not depend on
-/tmp. GRASS temporary files are created in the data base with the assumption
-that there will be enough space under the data base for large files.
+creates a unique temporary file or directory (<b>-f</b>) and prints the path.
+It is designed for shell scripts that need to use large temporary files.
+GRASS provides a mechanism for temporary files or directories that does not
+depend on /tmp. GRASS temporary files and directories are created in the
+GRASS GIS database with the assumption that there will be enough space for
```

Comment:
You can consider talking only about the project. See #3121 Phase 3, bullet point De-emphasize GISDBASE as its own thing from user perspective.

```suggestion
GRASS GIS project with the assumption that there will be enough space for
```

You anyway specify only $PROJECT below.